pkgs/top-level/all-packages.nix

```diff
@@ -8147,9 +8147,9 @@ with pkgs;
     else if libc == "musl" then
       musl
     else if libc == "msvcrt" then
-      windows.mingw_w64
+      if stdenv.hostPlatform.isMinGW then windows.mingw_w64 else windows.sdk
     else if libc == "ucrt" then
-      windows.mingw_w64
+      if stdenv.hostPlatform.isMinGW then windows.mingw_w64 else windows.sdk
```

Comment:
I think we'll want to make this just the CRT dir at some point, but that can be TODO for later

Reply:
Yeah, I'll put a note in the tracking issue.